### PR TITLE
routerrpc: revive legacy send and track rpcs

### PR DESCRIPTION
**Pull request by joostjager**
Soften the transition for `routerrpc` users to the V2 messages by creating a period of approx. 3 months  (the 0.11 release) to move over.

By transforming the current V2 messages to the legacy format, the amount of extra code and potential maintenance effort remains minimal. After the final release of 0.10, we have the option to cut it out again right away.

### CODE REVIEW

**Comment by alexbosworth** [comment]
Does send to route need to be included as well?

**Comment by Roasbeef** [comment]
I don't think this is necessary, what happened was the natural evolution of an experimental RPC. Post-mpp, we can expect things to change less. If we want to ease the transition of the users of this experimental RPC server (unclear how many there are, particularly given that we have no external documentation other than the protos), then we need to bridge the gap in terms of education, which we've discussed w.r.t he blog post series of how to use payment related RPCs post-mpp.

**Comment by cfromknecht** [comment]
If we got this route, maybe `Send`/`Pay` and `Track`? not a huge fan of the `V2` suffix myself. Otherwise the code changes make sense to me

**Comment by joostjager** [comment]
Closing PR for now. Can be reopened if we find out that we do need to remain backwards compatible with `routerrpc`

File: lnrpc/routerrpc/router_server_deprecated.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+// SendPayment attempts to route a payment described by the passed
+// PaymentRequest to the final destination. If we are unable to route the
+// payment, or cannot find a route that satisfies the constraints in the
+// PaymentRequest, then an error will be returned. Otherwise, the payment
+// pre-image, along with the final route will be returned.
+func (s *Server) SendPayment(request *SendPaymentRequest,
+	stream Router_SendPaymentServer) error {
+
+	if request.MaxShards > 1 {
```
Comment (Roasbeef) [inline lnrpc/routerrpc/router_server_deprecated.go:87]: 👍

**Pull request merged**